.travis.yml

```diff
@@ -41,6 +41,7 @@ before_deploy:
 - ls target/*.jar
 - cp target/otp-middleware-*.jar deploy/
 - cp target/otp-middleware-*.jar "deploy/otp-middleware-latest-$BRANCH_CLEAN.jar"
+- cp target/generated-sources/openapi/openapi.json "deploy/otp-middleware-api-latest-$BRANCH_CLEAN.json"
```

Comment:
@binh-dam-ibigroup, I would prefer that in addition to identifying deployment artifacts by the branch name, that we also publish artifacts using the `git describe` à la datatools-server (it uses a maven plugin for this): 

https://github.com/ibi-group/datatools-server/blob/423ed4d5859f0184f4587080c83f600f4b4481de/pom.xml#L68-L71

May make more sense to just use git describe in Travis rather than using the maven plugin, so that we can re-use the variable for  other artifacts as well (e.g., these API docs). By the way, is anything done with this JSON file once deployed? Or does it just sit in the S3 bucket?

Reply:
Oh good point, I'll append the git commit hash after the branch to identify each build.
Deployment beyond the S3 bucket should be more selective, like on a merge to the dev or  master branch?